docs/READMEpt-br.md

```diff
@@ -0,0 +1,185 @@
+```
+
+O código acima retornará o seguinte:
+
+```shell
+19012600725504
+```
+
+### Gráficos
+
+Você também pode pedir ao PandasAI para desenhar um gráfico:
+
+```python
+df.chat(
+    "Plot the histogram of countries showing for each the gdp, using different colors for each bar",
+)
+```
+
+![Chart](images/histogram-chart.png?raw=true)
+
+Você pode salvar todos os gráficos gerados pelo PandasAI configurando o parâmetro `save_charts` como `True` no construtor do `PandasAI`. Por exemplo, `PandasAI(llm, save_charts=True)`. Os gráficos são salvos em `./pandasai/exports/charts`.
+
+### Múltiplos DataFrames
+
+Além disso, você também pode passar vários DataFrames para o PandasAI e fazer perguntas relacionadas a eles.
+
+```python
+import pandas as pd
+from pandasai import SmartDatalake
+from pandasai.llm import OpenAI
+
+employees_data = {
+    'EmployeeID': [1, 2, 3, 4, 5],
+    'Name': ['John', 'Emma', 'Liam', 'Olivia', 'William'],
+    'Department': ['HR', 'Sales', 'IT', 'Marketing', 'Finance']
+}
+
+salaries_data = {
+    'EmployeeID': [1, 2, 3, 4, 5],
+    'Salary': [5000, 6000, 4500, 7000, 5500]
+}
+
+employees_df = pd.DataFrame(employees_data)
+salaries_df = pd.DataFrame(salaries_data)
+
+
+llm = OpenAI()
+dl = SmartDatalake([employees_df, salaries_df], config={"llm": llm})
+dl.chat("Who gets paid the most?")
+```
+
+O código acima retornará o seguinte:
+
+```console
+Oh, Olivia gets paid the most.
+```
+
+Você pode encontrar mais exemplos no diretório de [exemplos](examples) directory.
+
+### ⚡️ Atalhos
+
+O PandasAI também fornece uma série de atalhos (beta) para facilitar a realização de perguntas aos seus dados. Por exemplo, você pode pedir ao PandasAI para limpar dados, manipular valores ausentes, gerar colunas novas e plotar histograma, respectivamente, com as funções `clean_data`, `impute_missing_values`, `generate_features`, `plot_histogram`, e muitas outras.
+
+```python
+# Limpeza de dados
+df.clean_data()
+
+# Imputação de valores ausentes
+df.impute_missing_values()
+
+# Geração de colunas para modelagem
+df.generate_features()
+
+# Plotar histograma
+df.plot_histogram(column="gdp")
+```
+
+Saiba mais sobre os atalhos [aqui](https://pandas-ai.readthedocs.io/en/latest/shortcuts/).
+
+## 🔒 Privacidade e Segurança
+
+Para gerar o código Python a ser executado, pegamos a parte inicial do dataframe, fazemos uma randomização (usando geração aleatória para dados sensíveis e embaralhamento para dados não sensíveis) e enviamos apenas essa parte.
+
+Além disso, se você deseja reforçar ainda mais sua privacidade, pode instanciar o PandasAI com `enforce_privacy = True`, o que não enviará a parte inicial (mas apenas os nomes das colunas) para o LLM.
+
+## 🤝 Contributing
+
+Contribuições são bem-vindas! Por favor, confira as tarefas pendentes abaixo e sinta-se à vontade para abrir um pull request.
+Para obter mais informações, consulte as [diretrizes de contribuição](CONTRIBUTING.md).
```

Comment:
The link to the contributing guidelines is `docs/CONTRIBUTING.md`, whereas the original English version is located at `CONTRIBUTING.md` in the main directory. This leads to a different experience, as initially, the content is not viewable through GitHub. Consider changing this for consistency.